core/cmd/local_client.go

```diff
@@ -44,7 +45,9 @@ import (
 	webPresenters "github.com/smartcontractkit/chainlink/core/web/presenters"
 )
 
-func initLocalSubCmds(client *Client, devMode bool) []cli.Command {
+var ErrProfileTooLong = errors.New("requested profile too long")
+
+func InitLocalSubCmds(client *Client, devMode bool, opts *chainlink.GeneralConfigOpts) []cli.Command {
```

Comment:
Could we avoid exporting this if we used a `helpers_test.go` to make it available just for `package cmd_test`?

Reply:
i didn't know that trick. i will try it.

if we remove the safe guard in the Before func below, then this test will go away all together